.changes/changed/2752.md

```diff
@@ -0,0 +1 @@
+Extended the `TransactionStatus` to support pre-confirmations.
```

Comment:
i think we can be a little verbose here because it's a change in the graphql api :) non blocking, feel free to address in follow up